modules/Pharmacogenetics.py

```diff
@@ -115,6 +115,54 @@ def generate_evidence(
     assert enriched_pharmgkb_df.count() >= pharmgkb_df.count(), "There are fewer evidence after processing."
     return enriched_pharmgkb_df
 
+def add_variantid_column(input_df: DataFrame) -> DataFrame:
+    """ Adds a variantId column to the dataset"""
+    output_df = (
+        input_df
+        # extract chr_pos_ref_allele1_allele2 elements
```

Comment:
I think I'd use an approach different than extracting because when the genotype contains ref,ref or is multiallelic the extraction is not accurate
```
+---------------+---+---+---+-------+-------+
|     genotypeId|chr|pos|ref|allele1|allele2|
+---------------+---+---+---+-------+-------+
|    1_123_G_C,G|  1|123|  G|      C|      G|
|  1_123_G_GG,GC|  1|123|  G|     GG|     GC|
|      1_123_A_A|   |   |   |       |       |
|     1_123_T_TT|   |   |   |       |       |
|2_123_C_A,G,T,C|   |   |   |       |       |
+---------------+---+---+---+-------+-------+
````
(chr, pos, ref shouldn't be empty)